3-UsingAPIs/Week1/test-reports/ex1-johnWho.fail.txt

```diff
@@ -0,0 +1,11 @@
+
+*** ESLint Report ***
+
+ex1-johnWho.js
+  10:1  warning  Remove commented-out code  hyf/no-commented-out-code
+  17:1  warning  Remove commented-out code  hyf/no-commented-out-code
+  19:1  warning  Remove commented-out code  hyf/no-commented-out-code
+  23:1  warning  Remove commented-out code  hyf/no-commented-out-code
```

Comment:
Does not ignore these warnings but act on them.

Reply:
Done.